test/smith/mzcompose.yml

```diff
@@ -23,7 +23,12 @@ services:
     entrypoint: /bin/bash
     environment:
       - RUST_LOG=smith-fuzz=debug,info
-    command: -c "wait-for-it materialized:6875 --
-      --materialized-host materialized
-      --query-count 100"
+    entrypoint:
```

Comment:
This has two entrypoints, which may technically work but is philosophically wrong, it's just overwriting the earlier map key.

Also if you take my request to use the `mzconduct` buildkite plugin we can get wait-for-it out of the docker image, delete the override of the entrypoint, and just have the command be `--materialized-host materialized`

Reply:
I don't know how to use the mzconduct plugin offhand is there an existing example I can refer to?